cmd/release-controller/http_changelog.go

```diff
@@ -40,6 +66,30 @@ func (c *Controller) renderChangeLog(w http.ResponseWriter, fromPull string, fro
 			return
 		}
 
+		op, err := c.releaseInfo.ReleaseInfo(toPull)
+		if err != nil {
+			ch <- renderResult{err: fmt.Errorf("could not get release info for pullSpec %s: %v", toPull, err)}
+			return
+		}
+
+		releaseInfo := releaseInfoConfig{}
+		if err := json.Unmarshal([]byte(op), &releaseInfo); err != nil {
+			ch <- renderResult{err: fmt.Errorf("could not unmarshal release info for pullSpec %s: %v", toPull, err)}
+			return
+		}
+
+		// TODO: There is an inconsistency with what is returned from ReleaseInfo (amd64) and what
+		//       needs to be passed into the RHCOS diff engine (x86_64).
+		architecture := ""
```

Comment:
always do `var architecture, archExtension string` instead of this when initializing a variable to the default value.